application/gqlgen/mutations.go

```diff
@@ -110,6 +111,43 @@ func (r *mutationResolver) CreateOrganizationDomain(ctx context.Context, input C
 	return domains, nil
 }
 
+// UpdateOrganizationDomain is the resolver for the `updateOrganizationDomain` mutation
+func (r *mutationResolver) UpdateOrganizationDomain(ctx context.Context, input CreateOrganizationDomainInput) ([]models.OrganizationDomain, error) {
+	cUser := models.GetCurrentUserFromGqlContext(ctx)
+	extras := map[string]interface{}{
+		"user": cUser.UUID,
+	}
+
+	var org models.Organization
+	if err := org.FindByUUID(input.OrganizationID); err != nil {
+		return nil, reportError(ctx, err, "UpdateOrganizationDomain.NotFound", extras)
+	}
+
+	if !cUser.CanEditOrganization(org.ID) {
+		err := errors.New("insufficient permissions")
+		return nil, reportError(ctx, err, "UpdateOrganizationDomain.Unauthorized", extras)
+	}
+
+	var orgDomain models.OrganizationDomain
+	if err := orgDomain.FindByDomain(input.Domain); err != nil {
+		return nil, reportError(ctx, err, "UpdateOrganizationDomain.NotFound", extras)
+	}
+
+	orgDomain.AuthType = *input.AuthType
```

Comment:
I'm not sure if this is needed, but you did it above, so may as well be consistent.
```suggestion
	orgDomain.AuthType = domain.ConvertStrPtrToString(input.AuthType)
```